priam/src/main/java/com/netflix/priam/identity/InstanceIdentity.java

```diff
@@ -267,12 +267,6 @@ private void populateInstancesByAvailabilityZoneMultiMap() {
             seeds.add(instancesByAvailabilityZoneMultiMap.get(loc).get(0).getHostIP());
         }
 
-        // Remove this node from the seed list so Cassandra auto-bootstrap will kick in.  Unless this is the only node
-        // in the cluster.
-        if (seeds.size() > 1) {
-            seeds.remove(myInstance.getHostIP());
-        }
-
```

Comment:
Can you explain a bit here? The node is essentially adding itself to the node now. Why is that a good thing?